dashboard/modules/site/index.js

```diff
@@ -0,0 +1,25 @@
+export default {
+  tasks(self) {
+    if (process.env.CI !== '1') {
```

Comment:
Do you think we need to give individuals who are modifying an existing project that already has tasks some guidance on this early return?

Reply:
Giving an example is too much I think. This should be a simple task, to merge two objects in a proper way. I can just mention that they can do that if a task exists.

Reply:
Please change this logic to use `...` to pull the `cypress-config` task in if appropriate, which makes it easier for the developer to see where to put their own tasks without redundancy:

```
return {
  ...(process.env.CI === 1) ? {
    'cypress-config': { ... }
  },
  // your project specific tasks here
};
```